backend/apis/parse-docket-pdf.utils.js

```diff
@@ -85,7 +85,17 @@ function parseCharges(lines, sections) {
 
   walkSection(lines, sections, "CHARGES", addDispositionToCharges);
 
-  return charges;
+  const chargesWithSeverity = charges.map(charge => {
+    const { description, rest } = charge;
+    const words = description.split(/\s/);
+    const splitIndex = words.findIndex(word => word.toUpperCase() !== word);
+    const offenseName = words.slice(0, splitIndex).join(" ");
+    const severity = words.slice(splitIndex).join(" ");
+    const charges = { ...rest, offenseName, severity };
```

Comment:
```suggestion
    const charges = { ...charge, offenseName, severity };
```

The offenseName and severity will override anything in the charge object, since they are after the `...charge` inside of the object.

Reply:
I noticed that right after I pushed. I apologize I fixed it right after I pushed. I add and commuted I just didn’t push when I get home I will push it up.

Reply:
Oh ok I see now what your saying. I will adjust and remove rest

Reply:
The issue I am having when I remove the rest arg is that when I spread charges along with offenseName & severity charge still includes description. Unless I am doing it wrong I will push up my code right now and maybe you can tell me if I misunderstood.
Thanks for the feedback!

Reply:
Ah I see. I didn't realize that we were trying to remove properties from the charge. Yeah let's go back to using the rest args like you have above ^. 👍